service/src/java/org/apache/hive/service/auth/HiveAuthFactory.java

```diff
@@ -111,6 +111,13 @@ public HiveAuthFactory(HiveConf conf) throws TTransportException {
     // hadoopAuth is not simple, it does not guarantee it is kerberos
     hadoopAuth = conf.get(HADOOP_SECURITY_AUTHENTICATION, "simple");
 
+
+    // With refactoring of the code to support dual mode "http" and "binary"
```

Comment:
Let's go with /* .... */  for multi-line comments.

Reply:
done